README.md

```diff
@@ -1,2 +1,102 @@
 # diffmin
-Prototype of automatic diff minimization, see https://github.com/INRIA/spoon/issues/3397
+
+In a software development process based on code reviews, the visualization of 
+the diff between the base, and the target branch is of utmost importance. It is
+known that spurious formatting changes pollute code reviews and lead to
+unnecessary communication and code review work. Today, the only solution for
+minimizing the diff is one that ignores white spaces. This project will aim to
+automatically minimize pull requests by keeping only the AST changes.
+
+## Getting Started
+
+This package is not published yet but one can build the project from source.
+
+### Prerequisites
+
+1. Java 11+ JDK
+2. Maven
+3. Git
+
+### Installing
+
+1. Clone the repository.
+    ```shell
+    git clone https://github.com/SpoonLabs/diffmin.git
+    ```
+2. Install dependencies.
+    ```shell
+    mvn install
+    ```
```

Comment:
The `install` phase does not install dependencies per se, it installs your project into the local Maven repository. It also executes almost all other Maven phases, which in turn download dependencies, but that's not really the goal of the `install` phase and can be accomplished e.g. by several other phases (e.g. the `compile` phase).

Diffmin is currently designed as a CLI application, as opposed to a library or framework, and so installing it for use in other projects doesn't make much sense to me. I would remove this, and merge the `### Installing` section with the `### Build locally` section (s.t. it says to first clone the repo, and the run compile and assembly).